src/shadowbox/server/manager_service.ts

```diff
@@ -52,6 +53,7 @@ interface RequestParams {
   name?: string;
   metricsEnabled?: boolean;
   quota?: AccessKeyQuota;
+  port?: number;
```

Comment:
port is a string in RequestParams

Reply:
We talked about it offline. Restify will almost certainly give us a string, not a number.